edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Diagnostics/publisher/EventHubMetricsUpload.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+namespace Microsoft.Azure.Devices.Edge.Agent.Diagnostics.Publisher
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Text;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+    using Newtonsoft.Json;
+
+    public class EventHubMetricsUpload : IMetricsPublisher
```

Comment:
Since we are now supporting event hub upload, it should also tag metrics with a guid as we do in the log analytics upload.